lib/components/Column/Column.css.js

```diff
@@ -0,0 +1,7 @@
+export default {
+  '.column': {
+    flexBasis: 0,
+    flexGrow: 1,
+    minWidth: 0
```

Comment:
It seems `flexBasis` work as the arbitrary `minWidth` under flexbox 😅

Reply:
Yeah this was a lesson we learnt from the [`seek-style-guide`](https://github.com/seek-oss/seek-style-guide/pull/350) about how `min-width` is handled by flex elements.